test/integration/model/update.test.js

```diff
@@ -35,6 +35,53 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         }));
     });
 
+    it('should not check for notNull Violation for undefined values', function() {
+      const ownerId = 2;
+      let accountRowId;
```

Comment:
Get rid of `accountRowId`, if you do `account.findOne` you will directly get our target instance, as there is `sync` in before each.